src/test/java/integration/GenericFailureTest.java

```diff
@@ -43,7 +57,7 @@ void genericFailureHasPrecedence() {
         assertThat(result.getGenericFailure().getExceptionMessage())
                 .isPresent()
                 .hasValueSatisfying(
-                        startsWithString("java.lang.RuntimeException: Some super error here\n" +
+                        startsWithString("java.lang.RuntimeException: Some super error here" + crlf +
```

Comment:
You could replace all of those changes with `System.lineSeparator()` - this returns `\r\n` on windows and `\n` on everything else.

https://docs.oracle.com/javase/8/docs/api/java/lang/System.html#lineSeparator--

Reply:
You're right. That is infinitely times better.